packages/odyssey-react/src/components/Text/Text.module.scss

```diff
@@ -0,0 +1,343 @@
+/*!
+ * Copyright (c) 2021-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+.root {
+  color: $text-body;
+  font-family: $body-font-family;
+  font-size: ($base-font-size / 16px) * 100%;
+  font-style: normal;
+  font-weight: 400;
+  letter-spacing: 0;
+  line-height: $base-line-height;
+  font-feature-settings: "lnum", "pnum";
+  text-decoration-skip-ink: all;
+
+  /* stylelint-disable selector-max-type, selector-max-class, selector-class-pattern */
+  abbr,
+  &.abbr {
+    border-block-end: 1px dashed $color-primary-dark;
+    text-decoration: none;
+  }
+
+  em,
+  &.em {
+    font-style: italic;
+
+    > em {
```

Comment:
what are these deeply nested direct descendant selectors doing for us here and on line  44-45 below? Are `em` or `strong` tags within other `em` or `strong` tags supposed to have additional styles applied? Can I see an example of that?

Reply:
`em` and `strong` allow for self-nesting, indicating a greater degree of emphasis or importance. i believe the spec allows for continued nesting, but we only style for the first two levels. use cases beyond that seem iffy.

Reply:
@edburyenegren-okta understood, does this styleing happen only if they are direct descendants? See the way the styles are written here.

It also seems like we could move some of our stylelint escaped rules up above to this are so that we can focus in on just the areas where we use deeply nested element type selectors.

Reply:
Both `strong` and `em` allow for an intermediary element because they accept phrasing content. Per the spec, both of these use cases are acceptable:

```html
<p>I don't <em>want</em> pizza, I <em>need <em>two</em></em> slices.</p>
```

```html
<p>I don't <em>want</em> pizza, I <em>need <dfn id="stuffed-crust-def"><em>stuffed-crust</em></dfn></em> Domino's.</p>
```

As you note, our styles should likely be less restrictive to support this second case. However, if we're going to support mixed-element styling for type content, we may want to ensure/document the results of combining some common semantics.

Reply:
> It also seems like we could move some of our stylelint escaped rules up above to this are so that we can focus in on just the areas where we use deeply nested element type selectors.

I have updated the stylelint escaped rules to disable on a case-by-case basis.